src/mongocxx/test/options/index.cpp

```diff
@@ -15,20 +15,24 @@
 #include "catch.hpp"
 #include "helpers.hpp"
 
+#include <bsoncxx/builder/stream/document.hpp>
 #include <bsoncxx/stdx/make_unique.hpp>
 #include <mongocxx/instance.hpp>
 #include <mongocxx/options/index.hpp>
 #include <mongocxx/stdx.hpp>
 
+using namespace bsoncxx::builder::stream;
 using namespace mongocxx;
-using namespace mongocxx::options;
```

Comment:
Note five lines below where we qualify the `index` type as `options::index`, even though we had a `mongocxx::options` using-directive in scope.  I made this change out of the belief that it is poor style to have a using-directive in scope unless it is used for the entire scope.